deepchem/models/torch_models/normalizing_flows_pytorch.py

```diff
@@ -0,0 +1,90 @@
+"""
+Normalizing flows for transforming probability distributions using PyTorch.
+"""
+
+import logging
+from typing import Sequence, Tuple
+
+import torch
+import torch.nn as nn
+
+logger = logging.getLogger(__name__)
+
+
+class Affine(nn.Module):
+  """Class which performs the Affine transformation.
+
+  This transformation is based on the affinity of the base distribution with
+  the target distribution. A geometric transformation is applied where
+  the parameters performs changes on the scale and shift of a function
+  (inputs).
+
+  Normalizing Flow transformations must be bijective in order to compute
+  the logarithm of jacobian's determinant. For this reason, transformations
+  must perform a forward and inverse pass.
+
+  Example
+  --------
+  >>>
+  >> import deepchem as dc
```

Comment:
All these should be `>>>` since otherwise test will not run. (Some examples leave `>>>` out because the example is known to not run on doctest)